src/ResourceManager/Sql/Commands.Sql/ManagedInstance/Cmdlet/NewAzureSqlManagedInstance.cs

```diff
@@ -0,0 +1,244 @@
+            "NewBySkuNameParameterSetParameter";
+
+        protected const string NewByEditionAndComputeGenerationParameterSet =
+            "NewByEditionAndComputeGenerationParameterSet";
+
+        /// <summary>
+        /// Gets or sets the name of the managed instance name.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 0,
+            HelpMessage = "SQL Database Managed Instance name.")]
+        [Alias("Name")]
+        [ValidateNotNullOrEmpty]
+        public string ManagedInstanceName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the resource group to use.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 1,
+            HelpMessage = "The name of the resource group.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public override string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the admin credential of the Managed instance
+        /// </summary>
+        [Parameter(Mandatory = false, HelpMessage = "The SQL authentication credential of the Maanged Instance.")]
+        [ValidateNotNull]
+        public PSCredential AdministratorCredential { get; set; }
+
+        /// <summary>
+        /// The location in which to create the Managed instance
+        /// </summary>
+        [Parameter(Mandatory = true,
+            HelpMessage = "The location in which to create the Managed instance")]
+        [LocationCompleter("Microsoft.Sql/managedInstances")]
+        [ValidateNotNullOrEmpty]
+        public string Location { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Managed instance Subnet Id
+        /// </summary>
+        [Parameter(Mandatory = true,
+            HelpMessage = "The Subnet Id to use for Managed instance creation")]
+        [ValidateNotNullOrEmpty]
+        public string SubnetId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Managed instance License Type
+        /// </summary>
+        [Parameter(Mandatory = true,
+            HelpMessage = "Determines which License Type of Sql Azure Managed instance to use")]
+        [PSArgumentCompleter(Constants.LicenseTypeBasePrice, Constants.LicenseTypeLicenseIncluded)]
+        public string LicenseType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Storage Size in GB for Managed instance
+        /// </summary>
+        [Parameter(Mandatory = true,
+            HelpMessage = "Determines how much Storage size to associate with Managed instance")]
+        [ValidateNotNullOrEmpty]
+        public int StorageSizeInGB { get; set; }
+
+        /// <summary>
+        /// Gets or sets the VCores for Managed instance
+        /// </summary>
+        [Parameter(Mandatory = true,
+            HelpMessage = "Determines how much VCores to associate with Managed instance")]
+        [ValidateNotNullOrEmpty]
+        public int Vcore { get; set; }
```

Comment:
Discussed it with powershell team and I believe we agreed it should be `VCores` with capital `C`.